sqlmesh/core/config/connection.py

```diff
@@ -348,11 +348,28 @@ def init(cursor: duckdb.DuckDBPyConnection) -> None:
                 except Exception as e:
                     raise ConfigError(f"Failed to load extension {extension['name']}: {e}")
 
-            for field, setting in self.connector_config.items():
-                try:
-                    cursor.execute(f"SET {field} = '{setting}'")
-                except Exception as e:
-                    raise ConfigError(f"Failed to set connector config {field} to {setting}: {e}")
+            if self.connector_config:
+                option_names = list(self.connector_config)
+                in_part = ",".join(["?" for _ in range(len(option_names))])
```

Comment:
```suggestion
                in_part = ",".join("?" for _ in option_names)
```
nit: this should achieve the same